rapids-4-spark-tools/README.md

```diff
@@ -1,78 +1,197 @@
-```
+```bash
 com.nvidia.spark.rapids.tool.profiling.ProfileMain.main(Array("/path/to/eventlog1"))
 ```
 
 For multiple event logs comparison and analysis:
-```
+```bash
 com.nvidia.spark.rapids.tool.profiling.ProfileMain.main(Array("/path/to/eventlog1", "/path/to/eventlog2"))
 ```
 
-### How to compile and use from command-line
-1. `cd $SPARK_HOME (Download Apache Spark if required)`
-2. `./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar /path/to/eventlog1`
-
-### Options
+### Use from spark-submit
+```bash
+$SPARK_HOME/bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain \
+<Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar \
+/path/to/eventlog1 /path/to/eventlog2
 ```
-$ ./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar --help
-
-Spark event log profiling tool
-
-Example:
-
-# Input 1 or more event logs from local path:
-./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar /path/to/eventlog1 /path/to/eventlog2
-
-# If event log is from S3:
-export AWS_ACCESS_KEY_ID=xxx
-export AWS_SECRET_ACCESS_KEY=xxx
-./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar s3a://<BUCKET>/eventlog1 /path/to/eventlog2
-
-# If event log is from hdfs:
-./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar hdfs://<BUCKET>/eventlog1 /path/to/eventlog2
 
-# Change output directory to /tmp. It outputs as "rapids_4_spark_tools_output.log" in the local directory if the output directory is not specified.
-./bin/spark-submit --class com.nvidia.spark.rapids.tool.profiling.ProfileMain  <Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar -o /tmp /path/to/eventlog1
+### Options
+```bash
+$SPARK_HOME/bin/spark-submit \
+--class com.nvidia.spark.rapids.tool.profiling.ProfileMain \
+<Spark-Rapids-Repo>/rapids-4-spark-tools/target/rapids-4-spark-tools_2.12-<version>.jar \
+--help
 
 For usage see below:
+
+  -c, --compare                   Compare Applications (Recommended to compare
+                                  less than 10 applications). Default is false
+  -g, --generate-dot              Generate query visualizations in DOT format.
+                                  Default is false
+  -n, --num-output-rows  <arg>    Number of output rows for each Application.
+                                  Default is 1000
   -o, --output-directory  <arg>   Output directory. Default is current directory
```

Comment:
currently this only supports local filesystem, would like to fix that if we have time

Reply:
Yes, i think we need to change the --help output later to match the README. but we can do it later.